src/components/SocialMediaLinks/SocialMediaLinks.stories.tsx

```diff
@@ -0,0 +1,242 @@
+      table: { disable: true },
+    },
+    className: { table: { disable: true } },
+    color: {
+      table: { disable: true },
+    },
+    id: { table: { disable: true } },
+    layout: {
+      table: { disable: true },
+    },
+    linksData: { table: { disable: true } },
+    showLabels: { table: { disable: true } },
+    size: {
+      table: { disable: true },
+    },
+  },
+  parameters: {
+    controls: {
+      hideNoControlsWarning: true,
+    },
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=77103-126004&t=Yx1GyAfBJyhuGIGs-4",
+    },
+    jest: ["SocialMediaLinks.test.tsx"],
+  },
+};
+export default meta;
+type Story = StoryObj<typeof SocialMediaLinks>;
+
+/*
+ *👇 Render functions are a framework specific feature to allow you control on how the component renders.
+ * See https://storybook.js.org/docs/react/api/csf
+ * to learn how to use render functions.
+ */
+// @todo Instead of setting all controls to null, perhaps I should just have no controls and suppress the warning
+//   about not being configured for controls.
+export const Default: Story = {};
+
+export const WithControls: Story = {
+  args: {
+    borders: "none",
+    color: "textDefault",
+    className: undefined,
+    id: undefined,
+    layout: "row",
+    linksData: [
+      { type: "facebook" },
+      {
+        type: "twitter",
+        url: "twitter.com/elsewhere",
+        labelText: "Alt Twitter",
+      },
+      {
+        type: "instagram",
+      },
+    ],
+    showLabels: true,
+    size: "small",
+  },
+  argTypes: {
+    borders: {
+      control: { type: "radio" },
+      options: borderTypeArray,
+      table: {
+        disable: false,
+        defaultValue: { summary: "none" },
+      },
+    },
+    className: {
+      control: true,
+      table: {
+        disable: false,
+      },
+    },
+    color: {
+      control: { type: "radio" },
+      options: colorTypeArray,
+      table: {
+        disable: false,
+        defaultValue: { summary: "textDefault" },
+      },
+    },
+    id: {
+      control: true,
+      table: {
+        disable: false,
+      },
+    },
+    layout: {
+      control: { type: "radio" },
+      options: layoutTypesArray,
+      table: {
+        disable: false,
+        defaultValue: { summary: "row" },
+      },
+    },
+    linksData: { table: { disable: false } },
+    showLabels: {
+      table: {
+        disable: false,
+        defaultValue: { summary: false },
+      },
+    },
+    size: {
+      control: { type: "radio" },
+      options: sizeTypeArray,
+      table: {
+        disable: false,
+        defaultValue: { summary: "small" },
+      },
+    },
+  },
+};
+
+export const LayoutVariations: Story = {
+  render: () => (
+    <VStack align="stretch" spacing="l">
+      <Heading id="heading-layout" level="two" text="Layout Variations" />
+      <Box>
+        <Heading id="heading-row-layout" level="four" text="Row (Default)" />
```

Comment:
Visually, it's okay to have headings with these sizes come after one another. Semantically, however, this is incorrect and headings should be hierarchical. In this case, these should be level "three" instead of "four".
Accessibility information: https://www.w3.org/WAI/tutorials/page-structure/headings/ and https://www.a11yproject.com/posts/how-to-accessible-heading-structure/